sympy/solvers/ode/tests/test_systems.py

```diff
@@ -805,6 +1094,7 @@ def test_linear_3eq_order1_type4_skip():
     #         C3*sin(sqrt(3)*Integral(t**3 + log(t), t)))*exp(Integral(-t**2 - sin(t), t)))]
 
     dsolve_sol = dsolve(eq1)
+    sol1 = [_simpsol(sol) for sol in dsolve_sol]
     # dsolve_sol = [eq.subs(C3, -C3) for eq in dsolve_sol]
     # assert all(simplify(s1.rhs - ds1.rhs) == 0 for s1, ds1 in zip(sol1, dsolve_sol))
```

Comment:
Can these comments be removed?

Reply:
Yes, I will remove it in the next commit.